core/src/main/java/org/apache/carbondata/core/indexstore/SegmentWrapperContainer.java

```diff
@@ -31,6 +31,9 @@
 
   private SegmentWrapper[] segmentWrappers;
 
+  public SegmentWrapperContainer() {
```

Comment:
why is this change required?

Reply:
`NoSuchMethodException` is thrown as the default instance is called from reflectionUtils.